packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -436,20 +457,6 @@ export const RemoteServerFactory = {
                 return
             }
 
-            // If the request has an X-Amz-Cf-Id header, log it now
-            // to make it easier to associated CloudFront requests
-            // with Lambda log entries. Generally we avoid logging
-            // because it increases the volume of log data, but this
-            // is important for log analysis.
-            const cloudfrontId = req.headers['x-amz-cf-id']
-            if (cloudfrontId) {
-                // Log the Express app request id plus the cloudfront
-                // x-edge-request-id value. The resulting line in the logs
-                // will automatically include the lambda RequestId, so
-                // one line links all ids.
-                console.log(`Req ${res.locals.requestId} for x-edge-request-id ${cloudfrontId}`)
```

Comment:
Just curious.. why are we no longer logging this cloudfront id?

Reply:
It's being replaced by the correlation Id in a separate PR https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1787

Since then the PR was merged in develop.